RELEASE.md

```diff
@@ -0,0 +1,25 @@
+# Release Process
+
+The process of creating a release is described in this document. Replace `X.Y.Z` with the version to be released.
+
+## 1. Create a `release-X.Y.Z` branch
+
+The `release-X.Y.Z` branch should be created from the desired tag in [helm repository](https://github.com/helm/helm.git).
+For example, to create a new branch based on helm tag v3.7.2 the following commands can be executed:
+
+```
+git remote add helm https://github.com/helm/helm.git
+git fetch helm
+git checkout tags/v3.7.2 -b release-3.7.2
+```
+
+## 2. Create a `fybrik-release-X.Y.Z` branch
+
+Based on the branch that was created in previous section, a new branch `fybrik-release-X.Y.Z` should be created 
```

Comment:
What do you mean? Should we do any changes in the original helm code ?

Reply:
Yes, for example in helm v3.7.2 the changes are https://github.com/fybrik/helm/commit/97ff5f5667c4040edf05e94a4d51ec6bbd5b44a2 but that could change based on the helm version.

Reply:
@roytman I added explicit link to a commit containing the additions.